doc/rst/technotes/index.rst

```diff
@@ -3,13 +3,75 @@
 Technical Notes
 ===============
 
-Contents:
+Technical Notes describe in-progress language features that are not
+yet stable enough to be in the language specification. Additionally, a
+Techinical Note can describe implementation details that can be ignored
+by most Chapel programmers.
+
+Interoperability
+----------------
+
+.. toctree::
+   :maxdepth: 1
+
+   C Interoperability <extern>
```

Comment:
Will these have to be explicitly updated if the title is updated in the doc itself?

Reply:
No, but if it isn't, the outline title won't match the doc title. (Which isn't the worst sort of error). But generally I would expect us to update the outline title and the doc title together.

Reply:
It might be worth making an action item for after the release to add a check on this, I can see it being easy to miss

Reply:
I created https://github.com/Cray/chapel-private/issues/1377 for that.